2996_reflection/reflection.md

```diff
@@ -3386,14 +3393,17 @@ Add a new paragraph between [expr.eq]{.sref}/5 and /6:
 
 * [*.#]{.pnum} If both operands are null reflection values, then they compare equal.
 * [*.#]{.pnum} Otherwise, if one operand is a null reflection value, then they compare unequal.
-* [*.#]{.pnum} Otherwise, if one operand is a reflection of a namespace alias, alias template, or type alias and the other operand is not a reflection of the same kind of alias, they compare unequal. [A reflection of a type and a reflection of an alias to that same type do not compare equal.]{.note}
-* [*.#]{.pnum} Otherwise, if both operands are reflections of a namespace alias, alias template, or type alias, then they compare equal if their reflected aliases share the same name, are declared within the same enclosing scope, and alias the same underlying entity.
-* [*.#]{.pnum} Otherwise, if both operands are reflections of a base class specifier, then they compare equal if they are reflections of the same base class specifier.
-* [*.#]{.pnum} Otherwise, if one operand is a reflection of a base class specifier and the other is not, then they compare unequal.
-* [*.#]{.pnum} Otherwise, if neither operand is a reflection of a value, then they compare equal if they are reflections of the same entity.
-* [*.#]{.pnum} Otherwise, if one operand is a reflection of a value and the other is not, then they compare unequal.
-* [*.#]{.pnum} Otherwise, if both operands are reflections of values, then they compare equal if and only if the reflected values are _template-argument-equivalent_ ([temp.type]{.sref}).
-* [*.#]{.pnum} Otherwise the result is unspecified. [For example, if the operands are results of calls to `data_member_spec`]{.note}
+* [*.#]{.pnum} Otherwise, if both operands represent entities, then the compare equal if and only if they are reflections of the same entity.
+* [*.#]{.pnum} Otherwise, if one operand represents an entity, then they compare unequal.
+* [*.#]{.pnum} Otherwise, if both operands represent variables, then the compare equal if and only if they are reflections of the same entity.
+* [*.#]{.pnum} Otherwise, if one operand represents a variable, then they compare unequal.
+* [*.#]{.pnum} Otherwise, if one operand represents a namespace alias or type alias and the other operand does not represent the same kind of alias, they compare unequal. [A reflection of a type and a reflection of an alias to that same type do not compare equal.]{.note}
```

Comment:
What happened to alias template?

Reply:
Ah - As far as I can tell, alias _templates_ are entities, even if their specializations are not - and thus covered by the blanket "entities" clause above.

[[https://eel.is/c++draft/basic.pre#3]](basic.pre):
> An [entity](https://eel.is/c++draft/basic.pre#def:entity) is a value, object, reference, structured binding, function, enumerator, type, class member, bit-field, template, template specialization, namespace, or pack[.](https://eel.is/c++draft/basic.pre#3.sentence-1)

Actually, I might ask about this in CWG on Mattermost - that sentence sort of implies that an alias template specialization is an entity as well.

Reply:
Can we just sidestep the whole question and put the aliases (including alias template) before entities? :) I think that'd also make it more obvious what's going on here.